cfgov/unprocessed/css/molecules/expandable.less

```diff
@@ -0,0 +1,272 @@
+        position: relative;
+        cursor: pointer;
+
+        border-top: 1px solid @gray-50;
+        background: @white;
+    }
+
+    &_target,
+    &_content-animated {
+        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    }
+
+    &_content-animated {
+        position: absolute;
+        bottom: 0;
+        padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+
+    &_content {
+        height: 0;
+        border-bottom: 1px solid @gray-50;
+        overflow: hidden;
+        position: relative;
+        background: @white;
+
+        &-animated {
+            position: relative;
+        }
+    }
+
+    &__expanded .m-expandable_content {
+        height: 100%;
+    }
+
+    // Show/hide cues
+    .m-expandable_cue-open {
+        display: block;
+    }
+
+    .m-expandable_cue-close {
+        display: none;
+    }
+
+    &__expanded {
+        .m-expandable_cue-open {
+            display: none;
+        }
+
+        .m-expandable_cue-close {
+            display: block;
+        }
+    }
+}
+
+.m-expandable {
+    // Modifiers
+    &__borders {
+        .m-expandable_target,
+        .m-expandable_content {
+            border: 1px solid @gray-50;
+        }
+
+        .m-expandable_target {
+            border-bottom: none;
+        }
+
+        .m-expandable_content {
+            border-top: none;
+        }
+    }
+
+    &__midtone {
+        .m-expandable_target,
+        .m-expandable_content {
+            border-color: @gray-50;
+            background: @gray-5;
+        }
+    }
+}
+
+
+/* topdoc
+  name: Expandable header elements
+  family: cf-molecules
+  codenotes:
+    - |
+      Pattern structure
+      -----------------
+      .m-expandable_header
+        .m-expandable_header-left.m-expandable_label
+        .m-expandable_header-right.m-expandable_link
+          .m-expandable_cue-open
+            .m-expandable_cue-label
+          .m-expandable_cue-close
+            .m-expandable_cue-label
+  patterns:
+  - name: .m-expandable_header (element)
+    codenotes:
+      - |
+        .m-expandable_header
+          .m-expandable_header-left.m-expandable_label
+          .m-expandable_header-right.m-expandable_link
+            .m-expandable_cue-open
+              .m-expandable_cue-label
+            .m-expandable_cue-close
+              .m-expandable_cue-label
+    notes:
+      - "Creates a full-width container to house information that is always
+         visible."
+      - "Combine .m-expandable_header with .m-expandable_target for a full-width
+         trigger."
+      - "Allows you to float information left and right."
+  tags:
+  - cf-molecules
+*/
+.m-expandable {
+    &_header-left {
+        float: left;
+        width: 78%;
+        text-align: left;
+    }
+
+    &_header-right {
+        float: right;
+        width: 22%;
+        text-align: right;
+    }
+
+    &_label {
+        font-size: unit(@expandable_label-font-size / @base-font-size-px, em);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/db8aa458b54a98cbfce49223aebeea7e336afe1e